src/Data/MongoDBMock/singleDogResults.js

```diff
@@ -2,7 +2,7 @@ const singleDogResults = [
   {
```

Comment:
use the file from development and refactor your code for working with obedienceClass: 0 or 1 or 2 or 3  rather than string 'Klasa 0' or 'Klasa 1'